client/flutter/ios/Podfile

```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
-# platform :ios, '9.0'
+platform :ios, '9.0'
```

Comment:
Do we have a reason for using 9.0 rather than 8.0

[Flutter support starts at 8.0](flutter.dev/docs/deployment/ios#review-xcode-project-settings)

Reply:
Xcode 12 only supports iOS 9.0 or above for on device debugging. Cocoapods also assumes iOS 9.0 for the platform if it's not specified. Perhaps we can support iOS 8.0 in the future but right now getting the first release out is more important.

https://developer.apple.com/documentation/xcode-release-notes/xcode-12-release-notes